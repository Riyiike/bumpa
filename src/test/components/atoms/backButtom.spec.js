import React from "react";
import { shallow } from 'enzyme'
import BackBtn from "../../../components/atoms/button/backButton";

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useLocation: () => ({
      pathname: 'localhost:3000'
    }),

    useNavigate: () => ({
        pathname: 'localhost:3000'
    })
}))

describe('test button', () => {
    it('test back button', () => {
        const btn = shallow(<BackBtn />)
        expect(btn).toMatchSnapshot();
    })
})