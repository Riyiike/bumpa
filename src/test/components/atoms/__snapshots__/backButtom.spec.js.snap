// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button test back button 1`] = `
<button
  className="bg-White dark:bg-DarkBlue p-[.3em] px-[1.2em] rounded-sm mb-9 flex gap-2  items-center text-sm drop-shadow-md "
  onClick={[Function]}
>
  <img
    alt="back button"
    className="dark:invert"
    src="backIcon.svg"
  />
  Back
</button>
`;
